website/pages/de/help.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+
+const CompLibrary = require('../../core/CompLibrary.js');
+
+const Container = CompLibrary.Container;
+const GridBlock = CompLibrary.GridBlock;
+
+function Help(props) {
+  const {config: siteConfig, language = ''} = props;
+  const {baseUrl, docsUrl, repoUrl} = siteConfig;
+  const docsPart = `${docsUrl ? `${docsUrl}/` : ''}`;
+  const langPart = `${language ? `${language}/` : ''}`;
+  const docUrl = doc => `${baseUrl}${docsPart}${langPart}${doc}`;
+
+  const supportLinks = [
+    {
+      content: `Weiter zur [Bedienungsanleitung.](${docUrl(
+        'introduction',
+      )})`,
+      title: 'Handbuch/Bedienungsanleitung',
+    },
+    {
+      content: `Hilf mit, das Handbuch zu verbessern, <br>und werde Unterstützer auf [GitHub](${repoUrl})`,
+      title: 'Mitmachen',
+    },
+    {
+      content: `Für weitere Hilfe und Support besuchen Sie die<br> [Avolites Website](https://www.avolites.com/software/latest-version). <br>Hilfe auf Deutsch gibt es auf [www.avolites.de](https://www.avolites.de)`,
+      title: 'Offizieller Avolites-Support',
+    }
+  ];
+
+  return (
+    <div className="docMainWrapper wrapper">
+      <Container className="mainContainer documentContainer postContainer">
+        <div className="post">
+          <header className="postHeader">
+            <h1>Weitere Links</h1>
+          </header>
+          <GridBlock contents={supportLinks} layout="threeColumn" />
```

Comment:
Is it worth putting these in the i18n file?

Reply:
I really don't know. I needed to add some words for the German website, and found this the way to go

Reply:
I did briefly try but it didn't seem to like translating content with links in it. I think you would have to do the sentence and the link separately. 

I figured this didn't really cause any harm for now and we can always change it if the help page gets more complicated